services/app-api/utils/validation/completionSchemas.ts

```diff
@@ -59,54 +50,41 @@ const validNumberRegex = /^\.$|[0-9]/;
 
 // NUMBER - Number or Valid Strings
 const numberSchema = () =>
-  string()
-    .test({
-      message: error.INVALID_NUMBER_OR_NA,
-      test: (value) => {
-        if (value) {
-          const isValidStringValue = validNAValues.includes(value);
-          const isValidNumberValue = validNumberRegex.test(value);
-          return isValidStringValue || isValidNumberValue;
-        } else return true;
-      },
-    })
-    .test({
-      test: (value) => !isWhitespaceString(value),
-      message: error.REQUIRED_GENERIC,
-    });
+  string().test({
+    message: error.INVALID_NUMBER_OR_NA,
+    test: (value) => {
+      if (value) {
+        const isValidStringValue = validNAValues.includes(value);
+        const isValidNumberValue = validNumberRegex.test(value);
+        return isValidStringValue || isValidNumberValue;
+      } else return true;
+    },
+  });
 
 // NUMBER NOT LESS THAN ONE
 export const numberNotLessThanOne = () =>
   string()
     .required(error.REQUIRED_GENERIC)
     .test({
-      test: (value) => !isWhitespaceString(value),
-      message: error.REQUIRED_GENERIC,
-    })
-    .test({
-      test: (value) => validNumberRegex.test(value!),
+      test: (value) =>
+        (validNumberRegex.test(value!) && parseFloat(value!) >= 1) ||
+        validNAValues.includes(value!),
       message: error.INVALID_NUMBER,
-    })
-    .test({
-      test: (value) => parseInt(value!) >= 1,
-      message: error.NUMBER_LESS_THAN_ONE,
     });
 
 // NUMBER NOT LESS THAN ZERO
 export const numberNotLessThanZero = () =>
   string()
     .required(error.REQUIRED_GENERIC)
     .test({
-      test: (value) => !isWhitespaceString(value),
-      message: error.REQUIRED_GENERIC,
-    })
-    .test({
-      test: (value) => validNumberRegex.test(value!),
+      test: (value) => {
+        if (!value) return true;
+        return (
+          (validNumberRegex.test(value!) && parseFloat(value!) >= 0) ||
+          validNAValues.includes(value!)
+        );
```

Comment:
accept empty responses (`.required()` will fail these as needed)
otherwise accept valid number >= 0 OR valid Not applicable value